shared-modules/glew/glew.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
You should add `shared-modules` as git submodule with `git submodule add https://github.com/flathub/shared-modules.git`.

Reply:
Studying how to do this...